iroh-resolver/src/resolver.rs

```diff
@@ -1090,6 +1103,12 @@ impl<T: ContentLoader> Resolver<T> {
             let mut resolved_path = vec![cid];
 
             for part in tail {
+                // TODO(ramfox): we currently add an empty string to the path.tail vec
+                // when the original path ends in a "/". This is causing errors because
+                // we attempt to index into the data using an empty string, which fails
+                if part.is_empty() {
+                    continue;
```

Comment:
why `continue` and not `break`?

Reply:
Might not be the end and skip only empty parts. Should never happen, but...

Reply:
cause 3am & asmir's suggestions for handling this are better than what I wrote :)